frontend/javascripts/oxalis/model/sagas/update_actions.ts

```diff
@@ -113,26 +114,8 @@ type UpdateVolumeTracingUpdateAction = {
     zoomLevel: number;
   };
 };
-export type CreateSegmentUpdateAction = {
-  name: "createSegment";
-  value: {
-    id: number;
-    anchorPosition: Vector3 | null | undefined;
-    name: string | null | undefined;
-    color: Vector3 | null;
-    creationTime: number | null | undefined;
-  };
-};
-export type UpdateSegmentUpdateAction = {
-  name: "updateSegment";
-  value: {
-    id: number;
-    anchorPosition: Vector3 | null | undefined;
-    name: string | null | undefined;
-    color: Vector3 | null;
-    creationTime: number | null | undefined;
-  };
-};
+export type CreateSegmentUpdateAction = ReturnType<typeof createSegmentVolumeAction>;
```

Comment:
I think we can use the same pattern for the other update actions, too, but I didn't want to hijack this PR further..